client/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const ClientConstants = Object.freeze({
+  MIN_MAX_FEE_PER_GAS: 5000000000n,
+  MAX_MAX_FEE_PER_GAS: 50000000000n,
```

Comment:
This seems too low for sepolia? maybe it's ok if we have better logs (see below)